packages/modelviewer.dev/examples/staging-and-camera-control.html

```diff
@@ -302,7 +302,60 @@ <h4>Event listeners can cooperate with camera-controls</h4>
       </div>
     </div>
   </div>
+  <div class="sample">
+    <div id="demo-container-8" class="demo"></div>
+    <div class="content">
+      <div class="wrapper">
+        <div class="heading">
+          <h2 class="demo-title">Add other interactions like rotating environment map</h2>
+          <h4>Use parameters with rest turntable to rotate only the environment light</h4>
+          <p>By using shift key plus left click mouse or touch pad drag, one can rotate
+			  the environment map to see how the environment background would cast light
+			  onto the object from different angles when it is rotated. Updating orbit to 
+			  new theta value and then reseting the theta value to stop turntable from 
+		  rotating. </p>
+        </div>
+        <example-snippet stamp-to="demo-container-8" highlight-as="html">
+          <template>
+<model-viewer id="envlight-demo" camera-controls src="../shared-assets/models/glTF-Sample-Models/2.0/DamagedHelmet/glTF/DamagedHelmet.gltf"  environment-image="../assets/whipple_creek_regional_park_04_1k.hdr" alt="A 3D model of an astronaut"></model-viewer>
+<script type="module">
+  const modelViewer = document.querySelector("#envlight-demo");
+	  
+  const updateOrbit = () => {
+  const orbit = modelViewer.getCameraOrbit(); 
+  modelViewer.jumpCameraToGoal();
+  requestAnimationFrame(() => updateOrbit());
+  var isLightChanging = false;
+  // Add the event listeners for mousedown, mousemove, and mouseup
+  modelViewer.addEventListener('mousedown', (event) => {
+    isLightChanging = true;
+  }, true);
+
+  modelViewer.addEventListener('mousemove', (event) => {
+	  
+  if (isLightChanging === true) {
+    if (event.shiftKey && event.which == 1) {
```

Comment:
You'll probably want to call `event.preventDefault()` in this `if` statement so that you don't get the normal camera motion.